classes/User/User.mjs

```diff
@@ -1,5 +1,5 @@
 import dbDriver from "../../database/driver.mjs"
-let err_out = Object.assign(new Error(), {"status":500, "message":"N/A"})
+let err_out = Object.assign(new Error(), {status: 500, message: "N/A"})
```

Comment:
I don't think this `status` or `message` are ever used, so this is sotr of just `new Error()`. Maybe it doesn't need to be here at all.